### PR TITLE
Fix timing PCLs by adding EDMToMeConverter

### PR DESCRIPTION
**Pull request by ChrisMisan**
**PR description:**
This PR follows #35874 and addresses the issue for the timing PCLs (TDC and Sampic).

**PR validation:** 

- PCL of TDC was validated against relval 1041
- PCL of Sampic was validated using the manual test included in `CalibPPS/TimingCalibration/test/`

### CODE REVIEW

**Comment by tvami** [comment]
test parameters:
 - workflows = 1041

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by tvami** [comment]
cc @tlampen @tocheng

**Comment by tvami** [comment]
Hi @ChrisMisan I still dont see the new plots in here
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-d770cc/20256/runTheMatrix-results/1041.0_RunExpressPhy2017F+RunExpressPhy2017F+TIER0EXPPPSCAL+ALCASPLITPPSCAL+ALCAHARVDPPSCAL/DQM_V0001_R000305064__Express__PCLTest__ALCAPROMPT.root

is it not supposed to be in this file?

**Comment by tvami** [comment]
In general the `dqmDir` of `AlCaReco/PPSDiamondSampicTimingCalibrationPCL` is missing in that ROOT file
Screen Shot 2021-11-04 at 16 38 16

**Comment by ChrisMisan** [comment]
@tvami 1041 workflow is for the HPTDC, not Sampic, Sampic PCL for the moment can only be tested manually. For the HPTDC I don't expect any change to the plots, this fix is apparently needed for the harvester to correctly load files produced by the workers #35874

**Comment by tvami** [comment]
I understand that the SAMPIC cannot be tested. I dont understand why you say that for the HPTDC you dont expect changes. But let me rephrase https://github.com/cms-sw/cmssw/pull/35988#issuecomment-961395474 --> the DQM plots dont exist, I dont understand why (for the HPTDC)

**Comment by tvami** [comment]
I can also ask the equivalent question of
why isnt there anything about TDC here
https://github.com/cms-sw/cmssw/blob/master/CalibPPS/TimingCalibration/plugins/PPSDiamondSampicTimingCalibrationPCLHarvester.cc#L278-L287

**Comment by tvami** [comment]
> In general the `dqmDir` of `AlCaReco/PPSDiamondSampicTimingCalibrationPCL` is missing in that ROOT file Screen Shot 2021-11-04 at 16 38 53

So correcting this msg
based on
https://github.com/cms-sw/cmssw/blob/master/CalibPPS/TimingCalibration/plugins/PPSTimingCalibrationPCLHarvester.cc#L152

I'd expect to have a dqm folder named `AlCaReco/PPSTimingCalibrationPCL`

**Comment by ChrisMisan** [comment]
Alright, maybe it'd be good to ask @forthommel but from what I understand TDC harvester isn't saving any plots so those intermediate files between worker->harvester are beeing dropped as temporary files: 
https://github.com/forthommel/cmssw/blob/282611ea01492ca2b6966726f8748cff519ae64b/CalibPPS/TimingCalibration/p

**Comment by tvami** [comment]
@forthommel any comments?

**Comment by tvami** [comment]
+alca
 * private communication confirmed https://github.com/cms-sw/cmssw/pull/35988#issuecomment-961431007
 * future development is to add plots to TDC
 * SAMPIC will (and can only be) test with data

**Comment by tvami** [comment]
@perrotta @qliphy this is essentially fully signed

**Comment by qliphy** [comment]
+1
